docs/modules/migrate/pages/dmt-start-migration-cluster.adoc

```diff
@@ -0,0 +1,27 @@
+= Start the Migration Cluster
+:description: The migration cluster is created by the custom Hazelcast distribution to migrate the data between your source and target clusters.
+
+{description}
+
+To start the migration cluster, complete the following steps:
+
+. Open a terminal
+. Navigate to the folder containing the extracted DMT package
+. Enter the following command:
++
+[source,shell]
+----
+HZ_NETWORK_PORT_PORT=5702 HZ_CLUSTERNAME=migration ./bin/hz start 
+----
+
+If the specified port is available, the cluster starts on that port. Otherwise, Hazelcast tries to find a free port as described in the xref:clusters:network-configuration.adoc#port[Port] section of the Networking topic. You can confirm the port used by the cluster in the logs displayed in your terminal.
+
+You can find the _migration.yaml_ file in the root folder of the DMT download package. If your logs show that the cluster starts on a different port to that specified in this file, you must update the `address` field to match the port number used.
+
+DMT uses this configuration file to connect to the migration cluster when running the migration.
+
+NOTE: The _migration.yaml_ file uses the same configuration options as the Hazelcast CLC. For further information on the options, refer to the https://docs.hazelcast.com/clc/latest/clc-config[Hazelcast CLC documentation^].
+
+== Disconnect Client Applications
+
+Don't forget to disconnect all client applications from your source cluster before you continue.
```

Comment:
Is this necessary?

Reply:
Debatable ;-)

I chose to include it for the following reasons:

* It is mentioned as a step in the High-level Process section, so it ought to be reflected in the How-to topics for completeness and to reduce cognitive load for people while reading
* When people use docs, they might not be thinking as they follow the steps and could inadvertently miss a step that would otherwise be in their mind
* It was raised by Hendy when she commented on the original single-page version of this information

Bottom line is, it is short enough, and clear enough, that people can easily skip it if they don't need the reminder/full picture; but, it provides enough support to cover the first two reasons given above for including it